content/attacks/posts/2023-04-13-Yearn-Finance.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2023-04-13
+target-entities: 
+- Yearn Finance
+entity-types:
+- DeFi
+- Yield farming
```

Comment:
Rewrite as `Yield Aggregator`, and try to keep entity types and attack types as in updated articles in future